qemu/tests/cfg/qemu_cpu.cfg

```diff
@@ -18,19 +18,26 @@
     auto_cpu_model = no
 
     # QEMU versions:
-    variants qemu_flavor:
-        - @unknown:
-        - rhel:
-            variants rhel_major_version:
+    variants:
+        - @qemu_flavor_unknown:
+        - qemu_flavor_rhel:
+            variants:
                 - 6:
-                    variants rhel_version:
+                    only Host_RHEL.6
+                    variants:
                         - 6_0:
                         - 6_1:
                         - 6_2:
                         - 6_3:
                         - 6_4:
-        - upstream:
-            variants qemu_version:
+                - 7:
+                    only Host_RHEL.7
+                    variants:
```

Comment:
some new cpu model support on rhel7 only, 
some old cpu model don't support on rhel6 and rhel7

 cpu_model_intel.486, cpu_model_intel.pentium, cpu_model_intel.pentium2, cpu_model_intel.pentium3:
no qemu_flavor_rhel.6..host_cpu_vendor_intel
no qemu_flavor_unknown..host_cpu_vendor_intel
no qemu_flavor_rhel.6..host_cpu_vendor_unknown